manifest.php

```diff
@@ -27,16 +27,16 @@
     'label' => 'LTI Delivery Tool Provider',
     'description' => 'The LTI Delivery Tool Provider allows third party applications to embed deliveries created in Tao',
     'license' => 'GPL-2.0',
-    'version' => '3.7.2',
+    'version' => '3.8.0',
     'author' => 'Open Assessment Technologies',
     'requires' => array(
         'generis' => '>=5.2.0',
         'tao' => '>=13.2.0',
         'taoDeliveryRdf' => '>=1.0',
         'taoLti' => '>=3.2.2',
         'taoLtiBasicOutcome' => '>=2.6',
-        'taoResultServer' => '>=3.4.0',
-        'taoDelivery' => '>=7.0.0'
+        'taoResultServer' => '>=4.2.0',
+        'taoDelivery' => '>=7.5.0'
```

Comment:
why do you require the this version of taoDelivery ?

Reply:
This PR requires https://github.com/oat-sa/extension-tao-delivery/pull/253
version of taoDelivery changed to `7.5.0` there